awx/main/models/ha.py

```diff
@@ -332,3 +332,60 @@ def on_job_create(sender, instance, created=False, raw=False, **kwargs):
         instance=Instance.objects.me(),
         unified_job=instance,
     )
+
+
+class UnifiedJobTemplateInstanceGroupMembership(models.Model):
+
+    order_with_respect_to = 'unifiedjobtemplate'
```

Comment:
We could probably obtain this from the field to obliviate the need for this parameter, probably `.m2m_reverse_field_name()` or something. But I'm sure we can get by without that.

Reply:
Or `cls.lower()` as you did in the migration, assuming you could find the "from" model class

Reply:
Uploaded a solution to this.